kani-compiler/src/kani_middle/points_to/points_to_analysis.rs

```diff
@@ -521,7 +521,7 @@ impl<'a, 'tcx> PointsToAnalysis<'a, 'tcx> {
             | Rvalue::ShallowInitBox(operand, _)
             | Rvalue::Cast(_, operand, _)
             | Rvalue::Repeat(operand, ..) => self.successors_for_operand(state, operand),
-            Rvalue::Ref(_, _, ref_place) | Rvalue::AddressOf(_, ref_place) => {
+            Rvalue::Ref(_, _, ref_place) | Rvalue::RawPtr(_, ref_place) => {
```

Comment:
Please add this PR: https://github.com/rust-lang/rust/pull/127679
to the issue description of upstream changes that make this toolchain update necessary.

Reply:
Done! Thanks for finding that